cli/tests/test_init_command.rs

```diff
@@ -71,11 +71,9 @@ fn read_git_target(workspace_root: &Path) -> String {
 #[test]
 fn test_init_git_internal() {
```

Comment:
These tests appear to be copied to test_git_init.rs. Maybe we can just remove them?

Reply:
It looks like all of them were dupes, so I just deleted them. Good catch!